app/src/main/java/com/futurice/freesound/feature/search/SearchViewModel.java

```diff
@@ -65,6 +63,8 @@
     @NonNull
     private final Subject<String> searchTermRelay = BehaviorSubject
             .createDefault(NO_SEARCH);
+    @NonNull
+    private final BehaviorSubject<Option<Throwable>> lastError = BehaviorSubject.create();
```

Comment:
We've updated our readme with new naming conventions. This one should be named `lastErrorOnceAndStream` and as a initial value it should have `Option.none()`